docs/content/Deployment/Platform-Serverless-AWS.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Deploy with Serverless Framework on AWS
+permalink: /deployment/platforms/serverless/aws
+category: Deployment
+subCategory: Platforms
+menuOrder: 2
+---
+
+This guide walks you through deploying Cube.js with the [Serverless
+Framework][link-sls] on [AWS][link-aws].
+
+<!-- prettier-ignore-start -->
+[[warning |]]
```

Comment:
@hassankhan Let's do a different warning here. We need to warn that cube store will not work and refresh worker will require an extra step.